CHANGELOG.rst

```diff
@@ -1,9 +1,8 @@
 Unreleased
 ----------
-* Drop support for Wagtail versions 4.2 and 5.0 (@katdom13)
-* Support for Wagtail 5.2 (@katdom13)
```

Comment:
**question:** Did Wagtail 5.2 support not make it into a previous release of this package? If not, we should still explicitly include this line

Reply:
Hi @alxbridge ,
I retained the previous release notes and put it under a new release. Thanks